include/experimental/__p1673_bits/blas1_dot.hpp

```diff
@@ -50,6 +50,7 @@ namespace experimental {
 inline namespace __p1673_version_0 {
 namespace linalg {
 
+// dot: no policy, with init value
```

Comment:
These comments are harmless, but generally unnecessary.  (The policy argument's template parameter is called `ExecutionPolicy`, and the init value is called `init`.)

Reply:
yes, it was more of a "description" rather than the exact wording, just my personal style, which I think it helps tracking visually code organization. Sorry for adding them without asking! Won't add from now on ;)

Reply:
It actually comes subconsciously